packages/compass-components/package.json

```diff
@@ -77,6 +77,7 @@
     "ace-builds": "^1.4.3",
     "mongodb-ace-autocompleter": "^1.1.1",
     "mongodb-ace-mode": "^1.8.0",
+    "mongodb-query-parser": "^2.4.6",
```

Comment:
This pulls in acorn, amongst other things. Are we okay with adding it as a dependency of this package? Should this be put into a separate package?

Reply:
I think it's sorta unavoidable and would be a thing at some point anyway. Also the benefits of having this stuff in one place outweigh any possible downsides